<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGameWidget</class>
 <widget class="QWidget" name="NewGameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Game</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="whatsThis">
      <string>Settings for the players in the game</string>
     </property>
     <property name="title">
      <string>Players</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="numOfPlayersLabel">
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;How many players will be in the game&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Players:</string>
        </property>
        <property name="buddy">
         <cstring>spinNumOfPlayers</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinNumOfPlayers">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>154</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPlayer1Name">
        <property name="whatsThis">
         <string>The name of the first player</string>
        </property>
        <property name="text">
         <string>Player 1:</string>
        </property>
        <property name="buddy">
         <cstring>playerOneName</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="playerOneName"/>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="playerOneHuman">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="whatsThis">
         <string>Should this player be controlled by a human</string>
        </property>
        <property name="text">
         <string>&amp;Human?</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="playerOneAiLevel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Easy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AlphaBeta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dabble</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dabble.nohash</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QDab client</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Knox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MCTS A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MCTS B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MCTS C</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelPlayer2Name">
        <property name="whatsThis">
         <string>The name of the second player</string>
        </property>
        <property name="text">
         <string>Player 2:</string>
        </property>
        <property name="buddy">
         <cstring>playerTwoName</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="playerTwoName"/>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="playerTwoHuman">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="whatsThis">
         <string>Should this player be controlled by a human</string>
        </property>
        <property name="text">
         <string>H&amp;uman?</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="playerTwoAiLevel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Easy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AlphaBeta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dabble</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dabble.nohash</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QDab client</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Knox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MCTS A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MCTS B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MCTS C</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelPlayer3Name">
        <property name="whatsThis">
         <string>The name of the third player</string>
        </property>
        <property name="text">
         <string>Player 3:</string>
        </property>
        <property name="buddy">
         <cstring>playerThreeName</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="playerThreeName"/>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="playerThreeHuman">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="whatsThis">
         <string>Should this player be controlled by a human</string>
        </property>
        <property name="text">
         <string>Hu&amp;man?</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="playerThreeAiLevel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Easy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AlphaBeta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dabble</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dabble.nohash</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QDab client</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Knox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MCTS A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MCTS B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MCTS C</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelPlayer4Name">
        <property name="whatsThis">
         <string>The name of the fourth player</string>
        </property>
        <property name="text">
         <string>Player 4:</string>
        </property>
        <property name="buddy">
         <cstring>playerFourName</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="playerFourName"/>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="playerFourHuman">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="whatsThis">
         <string>Should this player be controlled by a human</string>
        </property>
        <property name="text">
         <string>Hum&amp;an?</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QComboBox" name="playerFourAiLevel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Easy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AlphaBeta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dabble</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dabble.nohash</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QDab client</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Knox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MCTS A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MCTS B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MCTS C</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="whatsThis">
      <string>Settings for the game board</string>
     </property>
     <property name="title">
      <string>Game Board</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelWidth">
        <property name="whatsThis">
         <string>The number of squares the area is wide</string>
        </property>
        <property name="text">
         <string>Width:</string>
        </property>
        <property name="buddy">
         <cstring>spinWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinHeight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelHeight">
        <property name="whatsThis">
         <string>The number of squares the area is high</string>
        </property>
        <property name="text">
         <string>Height:</string>
        </property>
        <property name="buddy">
         <cstring>spinHeight</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="spinWidth" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="whatsThis">
      <string>Settings for the game</string>
     </property>
     <property name="title">
      <string>Game Settings</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="quickStartCheck">
        <property name="whatsThis">
         <string>Partially fill in the board automatically before the game starts</string>
        </property>
        <property name="text">
         <string>Quick start</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="aiThinkTime">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="aiThinkTimeLabel">
        <property name="text">
         <string>max. AI think time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>350</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinNumOfPlayers</tabstop>
  <tabstop>playerOneName</tabstop>
  <tabstop>playerOneHuman</tabstop>
  <tabstop>playerTwoName</tabstop>
  <tabstop>playerTwoHuman</tabstop>
  <tabstop>playerThreeName</tabstop>
  <tabstop>playerThreeHuman</tabstop>
  <tabstop>playerFourName</tabstop>
  <tabstop>playerFourHuman</tabstop>
  <tabstop>spinWidth</tabstop>
  <tabstop>spinHeight</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>playerOneHuman</sender>
   <signal>clicked(bool)</signal>
   <receiver>playerOneAiLevel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playerThreeHuman</sender>
   <signal>clicked(bool)</signal>
   <receiver>playerThreeAiLevel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playerTwoHuman</sender>
   <signal>clicked(bool)</signal>
   <receiver>playerTwoAiLevel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playerFourHuman</sender>
   <signal>clicked(bool)</signal>
   <receiver>playerFourAiLevel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
